backend/pom.xml

```diff
@@ -55,7 +55,6 @@
             <artifactId>postgresql</artifactId>
             <version>42.2.5</version>
         </dependency>
-
```

Comment:
It was really necessary that minus line? You should Ctrl + Alt + L (autoformat) your files always before commiting.